CHANGELOG.md

```diff
@@ -8,6 +8,12 @@
 - LIST/GRID viewtype fix when updateUrl param is false
 - Added onQueryRedirect method on options to handle the ridirect
 - Added openNewTab flag to load the landing page url in same tab or new tab
+- infinite scroll on custom element bug fix.
+
+
+### Changed
+- updated the version from js-core
```

Comment:
lets mention the change that came in the new version instead of this generic message coz this repo is public & users will not know about search js-core library

Reply:
okay